lib/react_client.dart

```diff
@@ -364,7 +356,6 @@ final ReactDartInteropStatics _dartInteropStatics = (() {
 
   return new ReactDartInteropStatics(
       initComponent: allowInterop(initComponent),
-      handleGetChildContext: allowInterop(handleGetChildContext),
```

Comment:
This should be restored since the new stuff will be getting added to `ReactDartInteropStatics2`, and the existing `ReactDartInteropStatics` impl will be deprecated with no functional changes.